code/game/objects/items/weapons/twohanded.dm

```diff
@@ -245,6 +236,15 @@
 			var/atom/throw_target = get_edge_target_turf(M, get_dir(src, get_step_away(M, src)))
 			M.throw_at(throw_target, 5, 1)
 
+/obj/item/twohanded/fireaxe/boneaxe  // Blatant imitation of the fireaxe, but made out of bone.
```

Comment:
You just moved it around, and now git blame would think you deleted the old one and added the new one.
Please move back to above energized.

Reply:
Moved back